knitout-to-kcode.js

```diff
@@ -1068,7 +1087,7 @@ function knitoutToPasses(knitout, knitoutFile) {
 				}
 			}
 			setLast(cs, d, n);
-		} else if (op === 'pause') {
+		}else if (op === 'pause') {
```

Comment:
Another weird spacing change.

Reply:
yeah, been working on a new computer and it seems like my editor decided to change my formatting config to auto format and be weirdly opinionated on random spacing styles that I don't care about (?)... Fixed it though, so shouldn't happen in the future.